codeframe/ui/server.py

```
@@ -853,6 +853,198 @@ async def get_project_issues(project_id: int, include: str = None):
     return issues_data
 
 
+# Blocker endpoints (049-human-in-loop)
+
+@app.get("/api/projects/{project_id}/blockers")
+async def get_project_blockers(
+    project_id: int,
+    status: str = None
```

Ruff (0.14.4)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)